src/ucp/rndv/rndv.h

```diff
@@ -25,6 +25,7 @@ typedef struct {
     ucp_request_hdr_t         sreq;     /* send request on the rndv initiator side */
     uint64_t                  address;  /* holds the address of the data buffer on the sender's side */
     size_t                    size;     /* size of the data for sending */
+    uint8_t                   mem_type; /* memory type */
```

Comment:
we cannot change RTS header because it will break wire compatibility
need to take mem_type from unpacked remote key

Reply:
@yosefe fixed. can you please review it?